### PR TITLE
Migrated monitoring examples from notion.

### PR DESCRIPTION
**Pull request by petrpatek**
Rewrote the images link to follow the asset structure.

Next steps:
- Create an intro section.
- Consult if the images are correctly linked.
- Improve metadata

### CODE REVIEW

**Review by mnmkng — COMMENTED** [review]
Looks good to me. I'd only rename the images titled `Snimek_obrazovky...` to something more representative of what is shown in the image. Or just with the title of the example it belongs to with a number added.

**Review by mnmkng — COMMENTED** [review]
Thanks @petrpatek  for kicking this off! What will be the structure of the monitoring docs in the Docs? 

We should have an "intro page" about monitoring in general, which introduces the concept of monitoring, explains why it is important and useful and that everyone should monitor their actors' performance.

Next we introduce the monitoring suite, as a tool to do monitoring and the examples. I also think that the examples should be on separate pages so that they're easily accessible from the menu.

Can @m-murasovs or @davidjohnbarton help drafting the copy? The examples are done (pending proofreading), so only the intro texts for humans need to be drafted. I think @petrpatek got the general gist of it. It just needs a touch of a writer.

**Comment by m-murasovs** [comment]
Monitoring first draft is on develop: https://docs.apify.com/monitoring?version=develop. Pls review.

Images don't work on develop for some reason, I'm going to try and fix this tomorrow

**Comment by mnmkng** [comment]
Looks good to me! A few ideas:

> Learn how to ensure your actors, tasks and storages are running as you intended.

Maybe that's only me, but I got lost in the English here. Especially the "are running" felt a bit wonky. I know it's correct, but it made me stop and think twice before I got it.

- I'd change the title to Monitoring (or What is monitoring?) instead of What is the monitoring suite?
- I'd flip the order of information on the main page. First I'd say what monitoring is in general, without mentioning the monitoring suite or anything technical. Just a general paragraph about the usefulness and importance of monitoring your actors and their results. Then I'd talk about the monitoring suite (under a Monitoring suite section perhaps).
- After the first general section on monitoring, I'd add some screenshot from the Monitoring suite dashboard to quickly give the users idea what this is about.

Otherwise I think it's cool. Let's see with the screenshots in place, but I really like how you pieced it together using the Readme and added a lot of links to various places. Thanks a lot Misha.

File: docs/monitoring.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+The [monitoring suite](https://apify.com/apify/monitoring) is a collection of Apify [actors]({{@link actors.md}}) that allows you to automate the monitoring of jobs you have running on the Apify [platform](https://apify.com).
+
+You can use it to monitor anything from a single actor to complex projects spanning multiple actors, [tasks]({{@link actors/tasks.md}}) or [storages]({{@link storage.md}}).
+
+## [](#how-does-it-work) How does it work?
+
+The monitoring suite is made up of a system of actors we call **checkers**. Each checker is programmed to monitor one thing, e.g. the items in a [dataset]({{@link storage/dataset.md}}) for duplicates and correct format.
+
+Based on your configuration, the suite will create [schedules]({{@link schedules.md}}) and [webhooks]({{@link webhooks.md}}) under your account and use those to run the checkers. Some checkers will create visualizations of how your data progress over time, others will send [notifications](#notofications) about failed runs.
```
Comment (petrpatek) [inline docs/monitoring.md:20]: We have checkers and reporters. The checkers do the testing/data collecting part, but the once responsible of sending notifications and creating the visualization are reporters.

File: docs/monitoring/check_data_quality.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+## [](#use-case) Use case
+
+You want regularly scrape product data using a single scraper.
+
+You need:
+
+- Data to always be in the [correct format](#validate-data).
+- Alerts if items are [duplicated](#check-for-duplicates).
+- [Notification]({{@link monitoring.md#notifications}}) when your scheduled run finishes successfully.
```
Comment (petrpatek) [inline docs/monitoring/check_data_quality.md:22]: Monitoring does not support sending a notification when the run finishes successfully. It only allows sending a notification when the run is TIMED_OUT or FAILED.

**Review by petrpatek — COMMENTED** [review]
Thanks a lot. It looks great! 🎉

**Comment by m-murasovs** [comment]
I've fixed the develop branch images bug, so if you lads could check how it looks with images, that would be cool

https://docs-gamma-securitybyobscurity.apify.com/monitoring?version=develop

**Comment by petrpatek** [comment]
Looks good to me 👍

**Review by mnmkng — APPROVED** [review]

**Pull request merged**